docs/troubleshooting-guide/ldap.md

```diff
@@ -48,18 +59,23 @@ networks:
     driver: bridge
 ```
 
-If you don't pass environment variables to Metabase and you want to configure the environment manually, you can go to Admin->Settings->Authentication-> LDAP configuration and enter the following values:
+If you don't pass environment variables to Metabase and you want to configure the environment manually, you can go to the Admin Panel, selectin "Settings", select "Authentication", and then select "LDAP Configuration" and enter the following values:
 
-USERNAME OR DN: `cn=admin,dc=example,dc=org`
-PASSWORD: `adminpassword`
-USER SEARCH BASE: `ou=users,dc=example,dc=org`
-USER FILTER: `(&(objectClass=inetOrgPerson)(|(uid={login})))` // you can leave the default value which will look for the user id both in the uid or email field
-GROUP SEARCH BASE: `cn=readers`
+- `USERNAME OR DN`: `cn=admin,dc=example,dc=org`
+- `PASSWORD`: `adminpassword`
+- `USER SEARCH BASE`: `ou=users,dc=example,dc=org`
+- `USER FILTER`: `(&(objectClass=inetOrgPerson)(|(uid={login})))`
+- `GROUP SEARCH BASE`: `cn=readers`
+
+For the `USER FILTER`, you can leave the default value, which will look for the user ID in both the `uid` or `email` field.
 
 ### Related software for troubleshooting
 
-If you run into an issue, check that you can login and use your LDAP directory with software like [Apache Directory Studio](https://directory.apache.org/studio/). You can use Directory Studio to see the whole LDAP tree and view the logs of your LDAP application to see the queries run.
+If you run into an issue, check that you can login to your LDAP directory and issue queries using software like [Apache Directory Studio][apache-directory-studio]. It will let you see the whole LDAP tree and view the logs of your LDAP application to see queries run.
 
 ### Current limitations
 
-- When using Metabase Enterprise with a MySQL database and LDAP enabled, make sure that you disable the sync of binary fields from your LDAP directory by using the `MB_LDAP_SYNC_USER_ATTRIBUTES_BLACKLIST` environment variable, as you may hit the 60K field size limitation of the text field in MySQL, which will prevent the creation or log-in of your users.
+- When using Metabase Enterprise with a MySQL database and LDAP enabled, make sure that you disable synchronization of binary fields from your LDAP directory by using the `MB_LDAP_SYNC_USER_ATTRIBUTES_BLACKLIST` environment variable. If you do not, you may hit the 60K field size limitation of the text field in MySQL, which will prevent you from creating users or those users from logging in.
```

Comment:
```suggestion
- When using Metabase Enterprise with a MySQL database and LDAP enabled, make sure that you disable synchronization of binary fields from your LDAP directory by using the `MB_LDAP_SYNC_USER_ATTRIBUTES_BLACKLIST` environment variable. Otherwise, you may hit the 60K field size limitation of the text field in MySQL, which will prevent you from creating users, or prevent those users from logging in.
```